config/default.py

```diff
@@ -306,6 +306,7 @@
 REDSHIFT_SCHEMA_EDL_EXTERNAL = 'External EDL SIS schema name'
 REDSHIFT_SCHEMA_EDL_EXTERNAL_EDW = 'External EDL EDW schema name'
 REDSHIFT_SCHEMA_EDL_EXTERNAL_STAGING = 'External staging EDL SIS schema name'
+REDSHIFT_SCHEMA_EOP_ADVISING_NOTES = 'External EOP Advising Notes schema name'
```

Comment:
During this work, I noticed`REDSHIFT_SCHEMA_EOP_ADVISING_NOTES` was missing from `default.py`.